design/MonitoringModeAPI.md

```diff
@@ -1997,3 +2015,166 @@ Returns the recommendation at a particular timestamp if it exists
     "status": "ERROR"
 }
 ```
+
+## Update Recommendations API
+
+Generate the recommendations for a specific experiment based on provided parameters.
+
+**Request Parameters**
+
+| Parameter         | Type   | Required | Description                                                           |
+|-------------------|--------|----------|-----------------------------------------------------------------------|
+| experiment_name   | string | Yes      | The name of the experiment.                                           |
+| intervalEndTime   | string | Yes      | The end time of the interval in the format "yyyy-MM-ddTHH:mm:sssZ".   |
```

Comment:
@msvinaykumar - Should intervalEndTime and intervalStartTime be interval_end_time and interval_start_time?